### ISSUE TITLE
the push command crash

### ISSUE THREAD

**Issue by yishuiliunian** [esrlabs/git-repo#52]
```
Traceback (most recent call last):
  File "/Users/baidu/Yohoo/.repo/repo/main.py", line 538, in <module>
    _Main(sys.argv[1:])
  File "/Users/baidu/Yohoo/.repo/repo/main.py", line 512, in _Main
    result = repo._Run(argv) or 0
  File "/Users/baidu/Yohoo/.repo/repo/main.py", line 185, in _Run
    result = cmd.Execute(copts, cargs)
  File "/Users/baidu/Yohoo/.repo/repo/subcmds/push.py", line 381, in Execute
    self.manifest.topdir, abort_if_user_denies=True)
```

**Comment by mputz86** [comment]
Thanks! Can you check if this was fixed by @flxo with #51  ? Otherwise, please reopen the issue :).

**Comment by yishuiliunian** [comment]
fixed
